### PR TITLE
Create optional serviceAnnotations value in helm chart

### PR DESCRIPTION
**Pull request by jwenz723**
### Issue

None

### Description

Add optional annotations to the Kubernetes Service created by the helm chart.  Annotations are helpful for [human service discovery](https://kubernetes.io/blog/2021/04/20/annotating-k8s-for-humans/). My organization requires particular annotations to be set on all provisioned Services, this change will allow me to add the required annotations.

### Checklist
- [ ] Added tests that cover your change (if possible)
- [x] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [x] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Comment by jwenz723** [comment]
@M00nF1sh can you take a look at this?

**Review by kishorj — APPROVED** [review]

File: helm/aws-load-balancer-controller/values.yaml [hunk 1]
```diff
@@ -73,6 +73,9 @@ updateStrategy: {}
   #   maxSurge: 1
   #   maxUnavailable: 1
 
+# serviceAnnotations contains annotations to be added to the provisioned Service resource
```
[the remark this answers is not in the document]
Comment (jwenz723) [reply helm/aws-load-balancer-controller/values.yaml]: Done.

**Pull request merged**